### PR TITLE
Fix #44 Reddit icon

### LINKED ISSUE #44: Update Reddit icon svg to new Reddit style
Current icon svg which is the old style Reddit logo
Screenshot 2023-02-03 at 6 33 28 am

New updated reddit logo
logo01

### PR DESCRIPTION
**Pull request by jslawler**
Also correct registered trademark: https://trademarks.justia.com/owners/reddit-inc-2512865/

### CODE REVIEW

File: dist/companies.json [hunk 1]
```diff
@@ -8317,8 +8317,8 @@
             "description": "We create touch points throughout the consumer buying cycle. Through our digital expertise, proprietary assets, and deeply integrated partnerships, we close the loop between consumers and the brands or services they need."
         },
         "reddit": {
-            "name": "reddit",
-            "websiteUrl": "http://reddit.com",
+            "name": "Reddit, Inc.",
+            "websiteUrl": "www.reddit.com",
```
Comment (jellizaveta) [inline dist/companies.json:8321]: full website address: `https://www.reddit.com/`
file for changes: `adguard_companies.json`

**Review by jellizaveta — CHANGES_REQUESTED** [review]

**Review by ameshkov — CHANGES_REQUESTED** [review]
The changes should only be made to `adguard_companies.json`

File: dist/adguard_companies.json [hunk 2]
```diff
@@ -61,6 +61,11 @@
             "websiteUrl": "https://www.netflix.com/",
             "description": "Netflix is a streaming service that offers a wide variety of award-winning TV shows, movies, anime, documentaries, and more on thousands of internet-connected devices."
         },
+        "reddit": {
+            "name": "Reddit, Inc.",
+            "websiteUrl": "https://www.reddit.com",
+            "description": "\\\"reddit is a source for what's new and popular on the web. Users like you provide all of the content and decide, through voting, what's good and what's junk.\\\""
```
Comment (Alex-302) [inline dist/adguard_companies.json:67]: Is `\\\"` necessary?

Comment (ameshkov) [reply dist/adguard_companies.json:67]: comes from whotracksme db

**Review by ameshkov — APPROVED** [review]

**Review by jellizaveta — APPROVED** [review]

**Pull request merged**